src/linux/backend-hid.cpp

```diff
@@ -898,6 +917,36 @@ namespace librealsense
             return sampling_frequency_name;
         }
 
+        std::string iio_hid_sensor::get_sensitivity_name() const
+        {
+            std::string sensitivity_name = "";
+            DIR * dir = nullptr;
+            struct dirent * dir_ent = nullptr;
+
+            // start enumerate the scan elemnts dir.
+            dir = opendir( _iio_device_path.c_str() );
+            if( dir == nullptr )
+            {
+                throw linux_backend_exception( rsutils::string::from()
+                                               << "Failed to open scan_element " << _iio_device_path );
+            }
+
+            // verify file format. should include in_ (input) and _en (enable).
+            while( ( dir_ent = readdir( dir ) ) != nullptr )
+            {
+                if( dir_ent->d_type != DT_DIR )
+                {
+                    std::string file( dir_ent->d_name );
+                    if( file.find( "hysteresis" ) != std::string::npos )
```

Comment:
Can you explain this line?

Reply:
The relevant value of the sensitivity we want to write to pass to FW is written in a file that his name consists of "hysteresis", therefore we are looking for the file that his name has hysteresis in it.

Reply:
so `file` is actually `file_name`
Can you add a comment in the code explaining this/.

Reply:
The "like" is misleading as I don't see ay difference.
What I normally do is "LIKE" if I agree and when I push changes I add "DONE" or "Pushed" comment.
This way I know you changed and pushed it

Reply:
Added comment in the beginning of the function explaining it